lib/src/disposable/disposable.dart

```diff
@@ -427,6 +428,28 @@ class Disposable implements _Disposable, DisposableManagerV3 {
     return null;
   }
 
+  void subscribeToDocumentEvent(String event, EventListener callback,
```

Comment:
Are these methods we would want to expose on the `DisposableManager` interface? These methods would not be available to `LifecycleModule` without doing so.